### PR TITLE
Generated API server

### PR DESCRIPTION
**Pull request by borysei**

### CODE REVIEW

**Review by yushni — COMMENTED** [review]
1. Move all API related files to api folder. 
2. Delete main.go and main_test.go, it is unnecessary. 
3. Please change the path to main in app.yaml. It is necessary for a successful deployment. At the end of the app.yaml add 

main: ./api/cmd/[MAIN_SERVER_NAME]

File: models/error400.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	"github.com/go-openapi/errors"
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+	"github.com/go-openapi/validate"
+)
+
+// Error400 error400
+//
+// swagger:model Error400
+type Error400 struct {
```
Comment (yushni) [inline models/error400.go]: Please do not write numbers in naming. Rename 400 as BadRequest and 404 as ErrorNotFound and 500 as InternalError

File: restapi/configure_dictionary.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+	//
+	// Example:
+	// api.Logger = log.Printf
+
+	api.JSONConsumer = runtime.JSONConsumer()
+
+	api.JSONProducer = runtime.JSONProducer()
+
+	api.WordsAddWordHandler = words.AddWordHandlerFunc(func(params words.AddWordParams) middleware.Responder {
+		id := new(uint64)
```
Comment (yushni) [inline restapi/configure_dictionary.go]: It's how you can do without any functions. 
`	id := uint64(1)`	

`return words.NewAddWordOK().WithPayload(&words.AddWordOKBody{ID: &id})`
It's way how you can do with swageer helpers 


`		return words.NewAddWordOK().WithPayload(&words.AddWordOKBody{ID: swag.Uint64(1)})
`

File: restapi/configure_dictionary.go [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+	})
+
+	api.WordsGetWordsHandler = words.GetWordsHandlerFunc(func(params words.GetWordsParams) middleware.Responder {
+		w := make([]*models.Word, *params.Limit)
+
+		return words.NewGetWordsOK().WithPayload(w)
+	})
+
+	api.WordsDeleteWordHandler = words.DeleteWordHandlerFunc(func(params words.DeleteWordParams) middleware.Responder {
+		id := new(uint64)
```
Comment (yushni) [inline restapi/configure_dictionary.go]: `		return words.NewDeleteWordOK().WithPayload(&words.DeleteWordOKBody{ID: &params.WordID})
`

File: restapi/configure_dictionary.go [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+		*id = params.WordID
+
+		return words.NewDeleteWordOK().WithPayload(&words.DeleteWordOKBody{ID: id})
+	})
+
+	api.WordsGetWordHandler = words.GetWordHandlerFunc(func(params words.GetWordParams) middleware.Responder {
+		w := models.Word{
+			ID:            params.WordID,
+			Origin:        swag.String("Угорщина"),
+			Samples:       make([]string, 2),
```
Comment (yushni) [inline restapi/configure_dictionary.go]: Please make all field filled. Make 2 elements in slice will provide 2 empty string inside. We have to make it more realistic. 
`
			Samples: []string{
				"first sample",
				"second sample",
			},
`

The same thing with Translation.

File: restapi/configure_dictionary.go [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+	api.JSONProducer = runtime.JSONProducer()
+
+	api.WordsAddWordHandler = words.AddWordHandlerFunc(func(params words.AddWordParams) middleware.Responder {
+		id := new(uint64)
+		*id = uint64(1)
+		return words.NewAddWordOK().WithPayload(&words.AddWordOKBody{ID: id})
+	})
+
+	api.WordsGetWordsHandler = words.GetWordsHandlerFunc(func(params words.GetWordsParams) middleware.Responder {
+		w := make([]*models.Word, *params.Limit)
```
Comment (yushni) [inline restapi/configure_dictionary.go]: Screen Shot 2020-06-22 at 13 06 19
Please return a documented response with fake data

File: api/models/errorg_internal_error.go [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+	"github.com/go-openapi/errors"
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+	"github.com/go-openapi/validate"
+)
+
+// ErrorgInternalError errorg internal error
+//
+// swagger:model ErrorgInternalError
+type ErrorgInternalError struct {
```
Comment (yushni) [inline api/models/errorg_internal_error.go]: Errorg ?

Comment (borysei) [reply api/models/errorg_internal_error.go]: My bad

**Review by yushni — APPROVED** [review]

**Pull request merged**